api/src/app.ts

```diff
@@ -196,6 +196,10 @@ export default async function createApp(): Promise<express.Application> {
 				limit: env['MAX_PAYLOAD_SIZE'] as string,
 			}) as RequestHandler
 		)(req, res, (err: any) => {
+			if (req.method !== 'GET' && req.headers['content-length'] && typeof req.headers['content-type'] !== 'string') {
```

Comment:
`req.headers['content-length']` ensures a body has been sent before enforcing the `Content-Type` be set. Covers the case for requests that do not require a body at all times (e.g. DELETE`).

NOTE: It will bypass this check when the `Content-Length` is `'0'`, I was not sure how strict we wish to be here